vms/platformvm/txs/staker_tx.go

```diff
@@ -48,9 +48,13 @@ type Staker interface {
 	// PublicKey returns the BLS public key registered by this transaction. If
 	// there was no key registered by this transaction, it will return false.
 	PublicKey() (*bls.PublicKey, bool, error)
-	StartTime() time.Time
 	EndTime() time.Time
 	Weight() uint64
-	PendingPriority() Priority
 	CurrentPriority() Priority
```

Comment:
This could probably could be renamed Priority() instead of current priority. But I'd wait for that to keep PR scope small